test/e2e/density.go

```diff
@@ -61,6 +77,8 @@ type DensityTestConfig struct {
 	InternalClientset internalclientset.Interface
 	PollInterval      time.Duration
 	PodCount          int
+	// What kind of resource we want to create
+	resource string
```

Comment:
Instead of having a "resource" here, we should try changing ```Configs``` from ```[]testutils.RCConfig``` to ```[]<something>```  and then call appropriate methods on those.